testsuite/tests/typing-layouts/hash_types.ml

```diff
@@ -308,21 +308,23 @@ Error: The type constructor "t#" expects 1 argument(s),
 (* Type recursion through unboxed versions *)
 
 type t = int
-and bad = t#
+and ok = t#
 [%%expect{|
-Line 2, characters 0-12:
-2 | and bad = t#
-    ^^^^^^^^^^^^
-Error: The type "t" has no unboxed version.
+type t = int
+and bad = t#
```

Comment:
something funny here - it looks like this test needs to be promoted?

Reply:
"string" is used now